docs/src/tutorials/tutorial_4.md

```diff
@@ -246,45 +262,59 @@ Case: We want to do an SCC calculation across a base and marginal model of `Mimi
 
 The beginning steps for this case are identical to those above. We first define the typical variables for a simulation, including the number of trials `N` and the simulation definition `sim_def`.  In this case we only define one random variable, `t2xco2`, but note there could be any number of random variables defined here.
 
-```julia
+```jldoctest tutorial4; output = false
 using Mimi
 using MimiDICE2010
+using Distributions
 
-# define your trial number
+# define your trial number and discount rates
 N = 1000000 
+discount_rates = [0.3, 0.5, 0.7]
 
 # define your simulation (defaults to Monte Carlo sampling)
 sd = @defsim begin
-    t2xco2 = MyDistribution()
+    t2xco2 = Truncated(Gamma(6.47815626,0.547629469), 1.0, Inf) # a dummy distribution
 end
+
+# output
+
+MCSData()
 ```
 
 Next, we prepare our post-trial calculations by setting up a `scc_results` array to hold the results.  We then define a `post_trial_function` called `my_scc_calculation` which will calculate the SCC for that run.
 
-```julia
+```jldoctest tutorial4; output = false
 scc_results = zeros(N, length(discount_rates))
 
 function my_scc_calculation(sim_inst::SimulationInstance, trialnum::Int, ntimesteps::Int, tup::Tuple)
-    base, marginal = sim_inst.models
+    base = sim_inst.models[1]
+    marginal = sim_inst.models[2]
     base_damages = base[:neteconomy, :DAMAGES]
     marg_damages = marginal[:neteconomy, :DAMAGES]
     for (i, df) in enumerate(dfs)
         scc_results[trialnum, i] = sum(df .* (marg_damages .- base_damages))
     end
 end
+
+# output
+
+my_scc_calculation (generic function with 1 method)
 ```
 
 Now that we have our post-trial function, we can proceed to obtain our two models and run the simulation.
 
 ```julia
 # Build the base model
-base = construct_dice()
+base = MimiDICE2010.get_model()
 
-#Build the marginal model, which here involves a dummy function `construct_marginal_dice()` that you will need to write
-marginal = construct_marginal_dice(year) 
+#Build the marginal model
+mm = MimiDICE2010.get_marginal_model(m, year = 2015)
 
 # Run
-si = run(sd, [base, marginal], N; trials_output_filename = "ecs_sample.csv", post_trial_func = my_scc_calculation)
+si = run(sd, [mm.base, mm.marginal], N; trials_output_filename = "ecs_sample.csv", post_trial_func = my_scc_calculation)
```

Comment:
@ckingdon95 I know this isn't quite right anymore, since this line `si = ...` fails. Can you take a look and see if there's an easy fix or if I should delete this example?

Reply:
Is it okay if I push to this branch? I think there are a couple things wrong. It looks like the line 311 should construct the marginal model from `base` not `m`, and I think there's also a problem in the `my_scc_calculation` function definition (there is no `:DAMAGES` variable in `:neteconomy`, we should be using the consumption levels for the calculation)

Reply:
@ckingdon95 yes go for it that would be great! I’ll handle the doctests parts after the code looks right.